README.md

```diff
@@ -3,10 +3,12 @@
 # Welcome to [Lunie.io](https://lunie.io)!
 
 [![CircleCI](https://circleci.com/gh/luniehq/lunie/tree/develop.svg?style=svg)](https://circleci.com/gh/luniehq/lunie/tree/develop)
+[![Build Status iOS](https://app.bitrise.io/app/727927d247d126ae/status.svg?token=gmn_bux2oTIdwkDY68KGcA&branch=master)](https://app.bitrise.io/app/727927d247d126ae)
```

Comment:
Just an observation that you will have two widgets next to each other, but no way to distinguish them visually.

Reply:
TRUE!